pkg/ddc/base/runtime.go

```diff
@@ -235,15 +232,12 @@ func (info *RuntimeInfo) SetupWithDataset(dataset *datav1alpha1.Dataset) {
 }
 
 // SetupFuseDeployMode setups the fuse deploy mode
-func (info *RuntimeInfo) SetupFuseDeployMode(global bool, nodeSelector map[string]string) {
-	// Since Fluid v0.7.0, global is deprecated.
-	info.fuse.Global = true
+func (info *RuntimeInfo) SetupFuseDeployMode(nodeSelector map[string]string) {
 	info.fuse.NodeSelector = nodeSelector
 }
 
 // GetFuseDeployMode gets the fuse deploy mode
-func (info *RuntimeInfo) GetFuseDeployMode() (global bool, nodeSelector map[string]string) {
-	global = info.fuse.Global
+func (info *RuntimeInfo) GetFuseDeployMode() (nodeSelector map[string]string) {
```

Comment:
I think the method name `GetFuseNodeSelector`  is more accurate. and no need for local variable nodeSelector.